lighthouse/routes/common/plates.py

```diff
@@ -43,34 +45,19 @@ def create_plate_from_barcode() -> FlaskResponse:
     if request_json is None or barcode is None:
         return bad_request("POST request needs 'barcode' in body")
 
-    accepted_plate_types = app.config["SS_UUIDS"]
-    if plate_type is not None and plate_type not in accepted_plate_types:
-        return bad_request(f"POST request 'type' must be from the list: {', '.join(accepted_plate_types)}")
+    if plate_type is None:
+        plate_type = SS_PLATE_TYPE_DEFAULT
 
-    try:
-        # get samples for barcode
-        (fit_to_pick_samples, count_fit_to_pick_samples, _, _, _) = get_fit_to_pick_samples_and_counts(barcode)
-
-        if not fit_to_pick_samples:
-            return bad_request(f"No fit to pick samples for this barcode: {barcode}")
-
-        body = create_post_body(barcode, plate_type, fit_to_pick_samples)
+    plate_configs = app.config["SS_PLATE_CONFIG"]
+    if plate_type not in plate_configs:
+        return bad_request(f"POST request 'type' must be from the list: {', '.join(plate_configs)}")
```

Comment:
plate_configs is a dict, not a list

Reply:
Yes perhaps I should be more explicit, though in Python, any iteration of a dictionary without specifying a method on it just iterates the keys.